src/Component/style.css

```diff
@@ -0,0 +1,46 @@
+#helo{
```

Comment:
Seperate the CSS into multiple files.
Prefer class based styling over id selector and inline styling. Class base selector gives you more control when it comes to override the styles.